bolt/include/bolt/Profile/BoltAddressTranslation.h

```diff
@@ -11,6 +11,7 @@
 
 #include "llvm/ADT/SmallVector.h"
 #include "llvm/ADT/StringRef.h"
+#include "llvm/Support/DataExtractor.h"
```

Comment:
Not sure why was this moved here

Reply:
`DataExtractor` type is used in `parseMaps` declaration